src/qibolab/__init__.py

```diff
@@ -75,3 +78,46 @@ def execute_qasm(circuit: str, platform, initial_state=None, nshots=1000):
     return QibolabBackend(platform).execute_circuit(
         circuit, initial_state=initial_state, nshots=nshots
     )
+
+
+def get_available_platforms() -> list[str]:
+    """Returns the platforms found in the $QIBOLAB_PLATFORMS directory."""
+    return [
+        d.name
+        for d in get_platforms_path().iterdir()
+        if d.is_dir()
+        and Path(f"{os.environ.get(PLATFORMS)}/{d.name}/platform.py") in d.iterdir()
+    ]
+
+
+class MetaBackend:
+    """Meta-backend class which takes care of loading the qibolab backend."""
+
+    @staticmethod
+    def load(platform: str):
+        """Loads the backend.
+
+        Args:
+            platform (str): Name of the platform to load.
+        Returns:
+            qibo.backends.abstract.Backend: The loaded backend.
+        """
+        from qibolab.backends import QibolabBackend
+
+        return QibolabBackend(platform=platform)
+
+    def list_available(self) -> dict:
+        """Lists all the available qibolab platforms."""
+        available_platforms = {}
+        for platform in get_available_platforms():
+            try:
+                MetaBackend.load(platform)
+                available_platforms[platform] = True
+            except:
```

Comment:
I also don't see why this needs to be more complicated than just calling `get_available_platforms`. Also, global exceptions are not a good idea in general.